[PATCH] conditionally check libvirt

Upstream-Statue: Pending

check if libvirt is available only when a user wants to use libvirt

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 configure.ac | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 923498e..7206095 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4778,12 +4778,14 @@ then
 		with_libxml2="no (pkg-config doesn't know libxml-2.0)"
 	fi
 
-	$PKG_CONFIG --exists libvirt 2>/dev/null
-	if test "$?" = "0"
-	then
-		with_libvirt="yes"
-	else
-		with_libvirt="no (pkg-config doesn't know libvirt)"
+	if test "x$enable_libvirt" = "xyes"; then
+		$PKG_CONFIG --exists libvirt 2>/dev/null
+		if test "$?" = "0"
+		then
+			with_libvirt="yes"
+		else
+			with_libvirt="no (pkg-config doesn't know libvirt)"
+		fi
 	fi
 fi
 if test "x$with_libxml2" = "xyes"
-- 
1.9.1

